statistics/handle.go

```diff
@@ -145,10 +146,12 @@ func (h *Handle) Update(is infoschema.InfoSchema) error {
 }
 
 // GetTableStats retrieves the statistics table from cache, and the cache will be updated by a goroutine.
-func (h *Handle) GetTableStats(tblID int64) *Table {
-	tbl, ok := h.statsCache.Load().(statsCache)[tblID]
+func (h *Handle) GetTableStats(tblInfo *model.TableInfo) *Table {
+	tbl, ok := h.statsCache.Load().(statsCache)[tblInfo.ID]
 	if !ok {
-		return PseudoTable(tblID)
+		tbl = PseudoTable(tblInfo)
+		h.UpdateTableStats([]*Table{tbl}, nil)
```

Comment:
Can we not mix pseudo table with non-pseudo ones by adding `pseudoCache` in `Handle`?

Reply:
What's the benefit?

Reply:
So we don't need to load histograms for pseudo ones.

Reply:
But it won't load histograms for pseudo ones.